app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html class="no-js" lang="en">
 <head>
-  <link rel="shortcut icon" href="assets/icons/favicon.ico" />
+  <%= favicon_link_tag 'icons/favicon.ico' %>
```

Comment:
the "favicon_link_tag" already directs to "app/assets/images" and looks for it there, so we just need to add the icons directory :)  found in: http://stackoverflow.com/questions/22736831/rails-4-favicon-issue